variables.tf

```diff
@@ -23,6 +23,18 @@ variable "aws_load_balancer_controller_enabled" {
   description = "Enable ALB controller by default"
 }
 
+variable "cluster_encryption_config" {
+  type        = list(any)
+  default     = ["secrets"]
+  description = "Cluster Encryption Config Resources to encrypt, e.g. ['secrets']"
+}
+
+variable "cluster_kms_policy" {
```

Comment:
@sohanyadav how is this used? Is there an example or can we just have a default?